src/Tribe/Tickets_Handler.php

```diff
@@ -124,6 +124,47 @@ public function print_checkedin_totals() {
 	}
 
 	/**
+	 * Get the total capacity event.
+	 *
+	 * @param $event_id int (null)
+	 *
+	 * @since TBD
+	 *
+	 * @return int number of tickets ( -1 means unlimited )
+	 */
+	public function get_total_event_capacity( $event_id = null ) {
+		if ( null === $event_id ) {
+			global $post;
+			$event_id = $post->ID;
+		}
+
+		$capacity = 0;
+
+		$tickets = Tribe__Tickets__Tickets::get_event_tickets( $event_id );
+
+		if ( ! empty( $tickets ) ) {
+			foreach ( $tickets as $ticket ) {
+				$stock = $ticket->original_stock();
+
+				// Empty original stock means unlimited tickets, so we return -1 as a flag.
+				if ( empty( $stock ) ) {
+					// If one ticket is unlimited, total capacity is unlimited - bail with flag value
+					return -1;
```

Comment:
If we return here, folks can't use the hook that follows this loop: how about setting `$capacity` to `-1` and breaking out of the loop instead?